src/api/payment-requests/payment-requests.md

```diff
@@ -969,6 +969,102 @@ When you call release on a Pre Auth Payment Request any remaining funds that wer
 | 403    | {% break _ INVALID_PAYMENT_REQUEST_TYPE %} | The Payment Request is not related to a Pre Auth |
 | 403    | {% break _ PRE_AUTH_EXPIRED %}              | `preAuthExpiresAt` has passed                    |
 
+
+<a name="confirm"></a>
+### Make a confirmation against a Pre Auth Payment Request **EXPERIMENTAL**
+
+Confirmations require an `idempotencyKey` in order to ensure that a confirmation will not be processed twice. In the event of a duplicate confirmation request e.g. the payload matches an already completed confirmation, we will return a 200 with the related confirmation Payment Activity, and this confirmation will not be processed again. If an `idempotencyKey` is reused for a different confirmation request, we will return a 403. Note that the `idempotencyKey` only has to be unique for confirmations agains the same Payment Request.
+
+{% reqspec %}
+
+  POST '/api/payment-requests/${paymentRequestId}/confirm'
+  auth 'jwt'
+  path_param 'paymentRequestId', 'MhocUmpxxmgdHjr7DgKoKw'
+  example {
+    title 'Confirm Pre Authorization Payment Request'
+    body ({
+      "value": {
+        "amount": "6190",
+        "currency": "NZD",
+      },
+      "idempotencyKey": "e8df06e2-13a5-48b4-b670-3fd6d815fe0a",
+      "invoiceRef": "2022-08-03T16:56:50-06:00",
+      "lineItems": [
+        {
+          name: 'Coffee Grounds',
+          sku: 'GH1234',
+          qty: '1',
+          price: '4195',
+          tax: '15.00',
+        },
+        {
+          name: 'Centrapay Cafe Mug',
+          sku: 'SB456',
+          qty: '25',
+          price: '1995',
+          tax: '15.00',
+          discount: '199',
+          restricted: true,
```

Comment:
It looks like these restricted, productId and classification properties have not been implemented yet. Should we be publicly documenting them?